docker/py3-tools-ubi/pyproject.toml

```
@@ -8,7 +8,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "~3.10"
 cloudshare = "*"
-intezer-sdk = "*"
+intezer-sdk = "<=1.19.9"
```

using `^` promises we don't downgrade by mistake

```suggestion
intezer-sdk = "^1.19.9"
```